cli/azd/test/mocks/mock_context.go

```diff
@@ -31,7 +31,7 @@ func NewMockContext(ctx context.Context) *MockContext {
 	configManager := mockconfig.NewMockConfigManager()
 
 	mockexec.AddAzLoginMocks(commandRunner)
-	httpClient.AddDefaultMocks()
+	//mockarmresources.RegisterEmptyResources(httpClient)
```

Comment:
Just bringing this to your attention to either delete or uncomment.

Reply:
Oops, this should be deleted :) Was able to get the default mock removed in favor of explicit mock registration for `*GetProject` related tests